acquia_lift.module

```diff
@@ -461,6 +461,47 @@ function acquia_lift_percentage_validate($element, &$form_state) {
 }
 
 /**
+ * Retrieve the Acquia Lift Credentials.
+ * @return array
+ *   The value of the acquia_lift_account_info variable together with acquia
+ *   connector credentials.
+ */
+function acquia_lift_get_account_info() {
+  $account_info = variable_get('acquia_lift_account_info', array('public_key' => '', 'private_key' => ''));
+  if (variable_get('acquia_lift_v2_enabled', FALSE)) {
+    // @todo : Check here if Lift V2 is enabled by checking the subscription values
```

Comment:
This is not done yet. It would need an extra line to check if the status in the Acquia Subscription equals true. See https://www.evernote.com/l/ACyUuO-B-HpBUbip1wLYkU3zy6BoIiNEf2YB/image.png for the array that is returned from the Acquia Network.